### PR TITLE
Adapt the wrapper to new mapper configuration

### PR DESCRIPTION
**Pull request by boxanm**
See this PR: https://github.com/norlab-ulaval/norlab_icp_mapper/pull/15

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,40 +1,24 @@
...
-|             epsilon_d             |                                Fix error on the sensor distance (in meters).                                |              [0, ∞)              |                            0.01                            |
-|               alpha               |                       Probability of staying static given that the point was static.                        |              [0, 1]              |                            0.8                             |
-|               beta                |                      Probability of staying dynamic given that the point was dynamic.                       |              [0, 1]              |                            0.99                            |
-|               is_3D               |                       true when a 3D sensor is used, false when a 2D sensor is used.                        |          {true, false}           |                            true                            |
```
Comment (simonpierredeschenes) [inline README.md:32]: Shouldn't this parameter's documentation stay here?

Comment (boxanm) [reply README.md:32]: Idk what exact parameters you are referring to, but they were moved to the mapping_config file and are not node parameters anymore.

**Review by simonpierredeschenes — CHANGES_REQUESTED** [review]
There is a single comment, once it is addressed, I will approve and we can merge!
Thanks!

File: README.md (hunk 1 above)
Comment (simonpierredeschenes) [reply README.md:32]: I am talking about the is_3D parameter, I don't see it in the mapping_config

Comment (boxanm) [reply README.md:32]: Oh I see! Yeah, that parameter should be kept; on my side, GitHub showed me all the removed parameters, so it was super confusing. I'll put it back

**Review by simonpierredeschenes — APPROVED** [review]

**Comment by simonpierredeschenes** [comment]
Thanks a lot @boxanm !

**Pull request merged**